src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -1397,29 +1358,31 @@ static decimal INumber<decimal>.CreateTruncating<TOther>(TOther value)
             }
         }
 
-        static decimal INumber<decimal>.Clamp(decimal value, decimal min, decimal max)
-            => Math.Clamp(value, min, max);
+        /// <inheritdoc cref="INumber{TSelf}.Clamp(TSelf, TSelf, TSelf)" />
+        public static decimal Clamp(decimal value, decimal min, decimal max) => Math.Clamp(value, min, max);
 
-        static (decimal Quotient, decimal Remainder) INumber<decimal>.DivRem(decimal left, decimal right)
-            => (left / right, left % right);
+        /// <inheritdoc cref="INumber{TSelf}.DivRem(TSelf, TSelf)" />
+        public static (decimal Quotient, decimal Remainder) DivRem(decimal left, decimal right) => (left / right, left % right);
 
-        static decimal INumber<decimal>.Max(decimal x, decimal y)
-            => Math.Max(x, y);
-
-        static decimal INumber<decimal>.Min(decimal x, decimal y)
-            => Math.Min(x, y);
-
-        static decimal INumber<decimal>.Parse(string s, NumberStyles style, IFormatProvider? provider)
-            => Parse(s, style, provider);
+        /// <inheritdoc cref="INumber{TSelf}.Max(TSelf, TSelf)" />
+        public static decimal Max(decimal x, decimal y)
+        {
+            return DecCalc.VarDecCmp(in x, in y) >= 0 ? x : y;
+        }
 
-        static decimal INumber<decimal>.Parse(ReadOnlySpan<char> s, NumberStyles style, IFormatProvider? provider)
-            => Parse(s, style, provider);
+        /// <inheritdoc cref="INumber{TSelf}.Min(TSelf, TSelf)" />
+        public static decimal Min(decimal x, decimal y)
+        {
+            return DecCalc.VarDecCmp(in x, in y) < 0 ? x : y;
+        }
 
-        static decimal INumber<decimal>.Sign(decimal value)
-            => Math.Sign(value);
+        /// <inheritdoc cref="INumber{TSelf}.Sign(TSelf)" />
+        static decimal INumber<decimal>.Sign(decimal value) => Math.Sign(value);
```

Comment:
`decimal` already exposes `int Sign(decimal)`. We'll want to discuss in API review whether or not normalizing `Sign(T)` as being special and always returning `int` is desirable and how to reconcile the exposed API surface otherwise.